tck/src/main/java/org/reactivestreams/tck/TestEnvironment.java

```diff
@@ -81,7 +113,7 @@ public TestEnvironment(long defaultTimeoutMillis, long defaultNoSignalsTimeoutMi
    * @param defaultTimeoutMillis default timeout to be used in all expect* methods
    */
   public TestEnvironment(long defaultTimeoutMillis) {
-    this(defaultTimeoutMillis, defaultTimeoutMillis, false);
+    this(defaultTimeoutMillis, defaultTimeoutMillis, defaultTimeoutMillis);
```

Comment:
not sure if we need to default the new option to the same as it was before -- perhaps we can default it to a fourth of it or something? Note also the logic about getting the value from env https://github.com/reactive-streams/reactive-streams-jvm/pull/452/files#r278075241

Reply:
I'll defer to you here, but I think it would be surprising/undesirable to change existing behavior of a 1.x project.

Reply:
My thinking was that the only place this would change is that it may poll a few times and catch an error earlier... shouldn't really impact in other ways.

True through, we can stick to what it was and people can customize if they want to 🤔 Let's do that I guess, as you said.